runtime/bootstrap/index.ts

```diff
@@ -286,6 +290,12 @@ export async function injectExecutionContext({
     'executionContext.temporaryBlobStorage.storeUrl',
     temporaryBlobStorage.storeUrl.bind(temporaryBlobStorage),
   );
+
+  await injectAsyncFunction(
+    context,
+    'executionContext.syncState.getLatestRowVersions',
```

Comment:
Interesting, I guess `context.syncState.getLatestRowVersions` will technically be available at runtime even for non-syncs, since this code doesn't distinguish between whether this is a sync or not. I guess that's not terrible though as long as the TypeScript is correct.

Reply:
Edit: per the above, if `syncState` is passed as undefined we should only conditionally inject this.